api/v1beta1/azuremanagedmachinepool_types.go

```diff
@@ -483,6 +483,13 @@ type AzureManagedMachinePoolStatus struct {
 }
 
 // +kubebuilder:object:root=true
+// +kubebuilder:printcolumn:name="Cluster",type="string",JSONPath=".metadata.labels.cluster\\.x-k8s\\.io/cluster-name",description="Cluster to which this AzureManagedMachinePool belongs"
+// +kubebuilder:printcolumn:name="Ready",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].status"
+// +kubebuilder:printcolumn:name="Severity",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].severity"
+// +kubebuilder:printcolumn:name="Reason",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].reason"
+// +kubebuilder:printcolumn:name="Message",type="string",priority=1,JSONPath=".status.conditions[?(@.type=='Ready')].message"
+// +kubebuilder:printcolumn:name="Type",type="string",JSONPath=".spec.type",description="Type of Azure Identity"
```

Comment:
Was this meant for AzureClusterIdentity?

Reply:
I apologize, I didn't see that. No, it is for AzureManagedMachinePool.

Reply:
I think we should remove this then since AzureManagedMachinePool doesn't have a `spec.type` field.